configs/terraform/modules/slack-message-sender/slack-message-sender.tf

```diff
@@ -53,11 +53,15 @@ resource "google_cloud_run_service" "slack_message_sender" {
         }
         env {
           name  = "SLACK_CHANNEL_ID"
-          value = "C01KSP10MB5"
+          value = "C01KSP10MB5" # kyma-prow-dev-null
         }
         env {
           name  = "SLACK_RELEASE_CHANNEL_ID"
-          value = "C01KKPXCPK8"
+          value = "C01KKPXCPK8" # kyma-skr-release
+        }
+        env {
+          name  = "SLACK_TEAM_CHANNEL_ID"
+          value = "C01LGCBS196" # kyma-team
         }
```

Comment:
Can we have this env names more meaningfull? Something like:
PROW_DEV_NULL_SLACK_CHANNEL_ID
RELEASE_SLACK_CHANNEL_ID
KYMA_TEAM_SLACK_CHANNEL_ID